now.json

```diff
@@ -1,4 +1,19 @@
 {
+  "headers": [
+    {
+      "source": "/(.*)",
+      "headers": [
+        {
+          "key": "X-Content-Type-Options",
+          "value": "nosniff"
+        },
+        {
+          "key": "X-Frame-Options",
+          "value": "SAMEORIGIN"
+        }
```

Comment:
This disallows the site from being iframed right? Do we actually want to do that?

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Frame-Options

That's true.. what should I do? It's required by the above website :)

Reply:
We don't have any authentication, so there's no risk embedding it. I would not put this rule.